core/src/main/java/com/novoda/noplayer/PlayerSubtitlesView.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.noplayer;
+
+import com.google.android.exoplayer2.text.Cue;
+
+import java.util.List;
+
+public interface PlayerSubtitlesView {
```

Comment:
I agree. We should avoid using `ExoPlayer` or `MediaPlayer` specific code in the `no-player` part of this library.